aws/dms/tasks.yml

```diff
@@ -34,6 +34,7 @@ cron:
 - dashboard.contained_levels
 - dashboard.level_sources_multi_types
 - dashboard.paired_user_levels
+- dashboard.parental_permission_requests
```

Comment:
@carl-codeorg is this not under the `cron` job? are you saying it _should_ be under the `cron_pii`?...